tests/integration/targets/aci_fabric_interface_policy_group/tasks/main.yml

```diff
@@ -0,0 +1,292 @@
+# Test code for the ACI modules
+# Copyright: (c) 2023, Sabari Jaganathan (@sajagana)
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  fail:
+    msg: "Please define the following variables: aci_hostname, aci_username and aci_password."
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+# SET VARS
+- name: Set vars
+  set_fact:
+    aci_info: &aci_info
+      host: "{{ aci_hostname }}"
+      username: "{{ aci_username }}"
+      password: "{{ aci_password }}"
+      validate_certs: "{{ aci_validate_certs | default(false) }}"
+      use_ssl: "{{ aci_use_ssl | default(true) }}"
+      use_proxy: "{{ aci_use_proxy | default(true) }}"
+      output_level: '{{ aci_output_level | default("info") }}'
+
+- name: Ensure leaf_policy_group does not exist
```

Comment:
Think this configuration only applies for ACI 6.0, think we need version check to only execute for 6.0+ versions.

Reply:
Updated documentation for the "transceiver_policy_tdn" policy group.